protobufs/livekit_rtc_v2.proto

```diff
@@ -22,42 +22,43 @@ option ruby_package = "LiveKit::Proto";
 import "livekit_rtc.proto";
 import "livekit_models.proto";
 
-message Signalv2ClientMessage {
-  // monotonically increasing number (by 1) starting at some non-zero number (should not rollover)
-  uint32 message_id = 1;
-
-  // last remote message processed
-  uint32 last_processed_remote_message_id = 2;
-
+message Signalv2WireMessage {
   oneof message {
-    Fragment fragment = 3;
-    Signalv2ClientEnvelope envelope = 4;
-    ConnectRequest connect_request = 5;
+    Envelope envelope = 1;
+    Fragment fragment = 2;
   }
 }
 
-message Signalv2ClientEnvelope {
+message Envelope {
```

Comment:
making this generic, i. e. not making a server envelope and client envelope. Each side can just process whatever is right for them. The generic `Envelope` and `Fragment` is cleaner in my head.